corehq/apps/dump_reload/couch/dump.py

```diff
@@ -14,7 +14,6 @@
 from dimagi.utils.couch.database import iter_docs
 
 DOC_PROVIDERS = {
-    DocTypeIDProvider(['Location']),
```

Comment:
@snopoke as best as I could tell, this is used to determine what data to save to disk when a particular management command is run.  My assumption is that it's safe to just drop this entirely given that `Location` is being killed, and `SQLLocation` is being dumped anyways, does that sound correct?